devel/tweeny/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        mobius3 tweeny 2
+categories          devel graphics
+platforms           darwin
+license             MIT
+maintainers         {@scarface-one openmaintainer}
```

Comment:
This line is not correct. `openmaintainer` needs to be outside of the braces and your (obfuscated) email address added inside.